site/src/components/pages/user/Tabs.tsx

```diff
@@ -1,5 +1,7 @@
 import { VFC, FC } from "react";
 import { Box, Tabs, Typography, BoxProps, Tab } from "@mui/material";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
@teenoh can confirm, but you won't have to do this if you use our `Link` with the `noLinkStyles` prop.

Reply:
AFAIU, our Link component renders the child anchor tag, which we don’t want here. Judging by its code but I may be missing something here. All we want from Link around Tab is the right href and the right ctrl+click logic.

Reply:
Happy to update the diff if needed.

Reply:
The only reason we don't want the anchor tag here is because `Tab` is already rendered as one. If we passed `component: "span"` to `Tab`, we could let the custom `Link` render its anchor. 

But it would also need to accept children and put them within the anchor.

Reply:
This might not work if Tabs component expects Tab as a direct DOM child. Styles may break. So in the end it may end up more complicated than with next/link.

Reply:
The alternative is to remove `<Anchor ref={ref} {...other} />` from our link - NextJS will automatically insert an anchor tag if the children of `<NextLink>` is just plain text. If it is any other element, it will pass them in as-is. I believe this is the behavior we want. Then, we can pass `Tab` to our Link component, and it'll just pass it through.

Reply:
Remove it and replace it with the children

Reply:
It does: https://github.com/vercel/next.js/blob/ff6b75027b98ffc379282cb807551d1b5e9e7732/packages/next/client/link.tsx#L224

Reply:
Interesting! That might be a legacy feature because it’s not even mentioned in the docs. It won’t work in our case anyway because `children` in our case is `<Tab />`, not a `string` 🤔 

I am not saying that we can’t use our custom Link component, but I’m not sure I see the value in doing this. AFAIU, we’ll need to add some new prop that would just toggle a plain mode, i.e. fallback to standard `next/link` behaviour.

What we can do instead is consider creating a new component that would combine `Tabs` / `Tab` with `next/link`. We can explore this when we need to support tabs with links on more than one page.

Reply:
I was suggesting that if we replace `Anchor` in our own `Link` with just passing `children`, we would not need to change anything else. We could just pass `Tab` to it, since `Tab` already renders as an anchor.

But I don't think it's super important if there are other constraints/issues in doing so.

Reply:
I tried replacing `Anchor` with `children`, but it made `Link` even more convoluted. To save us from using `no-restricted-imports` in multiple files, I created a new file called `BaseLink` which is a minimal wrapper around `next/link`. It is only responsible for opening all external URLs in a new tab.